package.json

```diff
@@ -6,6 +6,7 @@
   "files": [
     "index.js"
   ],
+  "type": "module",
```

Comment:
this allows us to use es6 imports etc (instead of requires) within our files. i don't believe it will have any other impact?